Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM elixir:1.9-alpine as builder
+
+ENV MIX_ENV prod
+
+RUN apk add --no-cache build-base curl
+RUN mix local.hex --force
+RUN mix local.rebar --force
+RUN mkdir -p /mini_repo
+
+WORKDIR /mini_repo
+
+ADD config  ./config/
+ADD lib  ./lib/
+ADD priv ./priv/
+ADD rel ./rel/
+COPY mix.exs mix.lock ./
```

Comment:
This could take better advantage of docker layer caching by doing something like the following untested code.

``` dockerfile
# Deps
COPY config/config.exs config/prod.exs ./config/
COPY mix.exs mix.lock ./
RUN mix deps.get --only=prod && mix deps.compile

# App
COPY lib ./lib/
COPY priv ./priv/
RUN mix compile

# Release
COPY config/release.exs ./config/
RUN mix release
```

Reply:
yup, thanks @blatyo!